crates/red_knot_python_semantic/src/symbol.rs

```diff
@@ -107,6 +107,33 @@ impl<'db> Symbol<'db> {
             qualifiers,
         }
     }
+
+    /// Try to call `__get__(None, owner)` on the type of this symbol (not on the meta type).
+    /// If it succeeds, return the `__get__` return type. Otherwise, returns the original symbol.
```

Comment:
Worth adding a note and/or link here to say that this is part of our modelling of the descriptor protocol? A first-time reader of the codebase might wonder why we check for `__get__` here